### PR TITLE
implement esModule option

### PR DESCRIPTION
**Pull request by HerringtonDarkholme**
Related to https://github.com/kisenka/svg-sprite-loader/issues/55

If you think this is acceptable, I will add docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ module.exports = {
 * `angularBaseWorkaround` Adds workaround for issue with combination of `<base>` and History API which is [typical for Angular.js](https://github.com/angular/angular.js/issues/8934). Default is `false`.
 * `prefixize` isolates an image content by prefixing its `id`, `xlink:href` and `url(#id)` elements. Default is `true`.
 * `spriteModule` defines [custom sprite implementation](#custom-sprite-implementation) module path.
+* `esModule` whether to transiple module to es compatible format. When true, loader will produce `module.exports.__esModule = true; module.exports['default'] = svg`. Default is `false`. Useful for transipler other than Babel.
```
Comment (princed) [inline README.md]: 1. I would capitalise `es`
2. Typo in `transiple` and `transipler`

Comment (princed) [inline README.md]: 3. Backticks for `true` would be nice.

Comment (HerringtonDarkholme) [inline README.md]: Thanks! I have changed it.

Comment (princed) [inline README.md]: 👍

**Pull request merged**